modules/extension/app-schema/app-schema/src/main/java/org/geotools/data/complex/AppSchemaDataAccess.java

```diff
@@ -93,6 +93,14 @@
     private FilterFactory2 filterFac = CommonFactoryFinder.getFilterFactory2(null);
 
     /**
+     * Flag to mark non-accessible data accesses, which should be automatically
+     * disposed of when no longer needed by any accessible data access.
+     */
+    boolean hidden = false;
+
+    List<Name> relatedTypeNames = new ArrayList<>();
```

Comment:
@bencaradocdavies Indeed, it's a remnant from a discarded design, I'll
remove it tomorrow

Should I add a commit to the PR or rebase? I remember you advising against
rebasing a PR branch...
Il 10/set/2015 23:14, "Ben Caradoc-Davies" <notifications@github.com> ha
scritto:

> In
> modules/extension/app-schema/app-schema/src/main/java/org/geotools/data/complex/AppSchemaDataAccess.java
> <https://github.com/geotools/geotools/pull/956#discussion_r39214692>:
>
> > @@ -93,6 +93,14 @@
> >      private FilterFactory2 filterFac = CommonFactoryFinder.getFilterFactory2(null);
> >
> >      /**
> > +     * Flag to mark non-accessible data accesses, which should be automatically
> > +     * disposed of when no longer needed by any accessible data access.
> > +     */
> > +    boolean hidden = false;
> > +
> > +    List<Name> relatedTypeNames = new ArrayList<>();
>
> @ridethepenguin <https://github.com/ridethepenguin> this new member
> relatedTypeNames is not used anywhere and should be removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geotools/geotools/pull/956/files#r39214692>.
>